types/three/src/core/GLBufferAttribute.d.ts

```diff
@@ -5,13 +5,19 @@
 export class GLBufferAttribute {
     constructor(buffer: WebGLBuffer, type: number, itemSize: number, elementSize: 1 | 2 | 4, count: number);
 
+    /**
+     * @default ''
+     */
+    name: string;
```

Comment:
where is it set?

Reply:
Good point. For whatever reason the name property isn't initialized as an empty string in `GLBufferAttribute` even though both the other attribute types do that. I'll remove the comment.

Reply:
Another option would be to set the field as
```
    /**
     * @default undefined
     */
    name?: string;
```

Reply:
I agree that it's weird `GLBufferAttribute` doesn't have `name` on the Three.js side.
But, if it's not appeared in the Three.js source yet, it should not be added here I think.

> (even though this adds risk of further breakage if people are relying on using the name property on attributes...)

That's actually a good point.
I feel like we have to ask Three.js side about `GLBufferAttribute.name` first.

> Another option would be to set the field as
> 
> ```
>     /**
>      * @default undefined
>      */
>     name?: string;
> ```

I choose this if Three.js side says `GLBufferAttribute.name` should exist.

@mikkom Would you ask about this somewhere? I would choose [Discourse](https://discourse.threejs.org/).

Reply:
You did it on Three.js side, wooo! Thank you! Now it's valid to say the `.name`.